### DISCUSSION TITLE
Entrega de practica 1

### DISCUSSION THREAD

**Discussion by JulioRamnses** [Launch-X-Latam/MisionFrontEnd#105] [Q&A]
Hola Buenas Noches/Tardes/Dia

Quiero preguntar sobre lo que se entrega de la primera practica
![image](https://user-images.githubusercontent.com/114268025/193377950-74040811-77bc-4263-8be0-bc8a6a20239e.png)
de lo que se muestra en la imagen saben que es lo que se tiene que entregar??

estoy un poco perdido en esa parte, puesto que en los live solo sabemos como se hace el Wireframe y el boceto de este 

Disculpen las molestias y gracias  (•◡•) /

**Comment by JosafatJimenezB** [comment]
Todo lo que acabas de mencionar es lo que se entrega, ya que eso son los requerimientos que necesitas para poder comenzar con el proyecto

**Reply by maxkario** [reply]
Es decir, el wireframe unicamente?

**Reply by JosafatJimenezB** [reply]
todo lo que menciono el compañero

**Comment by samujm** [comment]
Hola!! a mi parecer tienes que ir llenando el documento de requerimientos (dentro del mismo vienen otros detallitos), crear diagramas de flujo, saber que grupo de personas objetivos tienes, crear el boceto o wireframe y el UI

**Comment by Amhedriel** [comment]
Buenas noches compañeros, esta pregunta ya se respondio en otras publicaciones, por favor lean todas las conversaciones antes de repetir una pregunta varias veces.

Ahora bien, un compañero respondio a una de las primeras conversaciones de este tema compartiendo un ejemplo de la Practica 1 en su repositorio de [GitHub](https://github.com/JosafatJimenezB/MissionFrontEnd "Práctica 1").

Como verán me parece que estuvo en un programa similar hace unos meses, saludos.
